### PR TITLE
USER STORY #19069: ADD TIME-STAMP LABEL INTO EVERY APPLICATION

### PR DESCRIPTION
**Pull request by rmagnum2002**
[``* `#19069` ADD TIME-STAMP LABEL INTO EVERY APPLICATION``](https://community.openproject.org/work_packages/19069)

### CODE REVIEW

**Comment by barhot** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -51,3 +51,7 @@ ONBUILD RUN su app -c 'cd /home/app/bundle-cache && \
 ONBUILD COPY / /home/app/webapp
 ONBUILD RUN cp -a /home/app/bundle-cache/.bundle /home/app/webapp
 ONBUILD RUN mkdir -p log tmp public && chown --recursive app log tmp public
+
+# save time-stamp in a file on docker build
+ONBUILD RUN echo $(/bin/date "+%Y-%m-%d %H-%M-%S") >  /home/app/webapp/public/build_timestamp.txt
+ONBUILD RUN chown app /home/app/webapp/public/build_timestamp.txt
```
Comment (linki) [inline Dockerfile]: looks ok, but: skip the last line and put the `echo` stuff be fore line `53`

Comment (linki) [inline Dockerfile]: whitespace after `>` line 56

**Comment by linki** [comment]
thanks @rmagnum2002

**Pull request merged**